### PR TITLE
#4 Build Info - develop service and endpoint

### PR DESCRIPTION
**Pull request by oehm-smith**
Endpoints are:

* /manifests/internal - geodesy projects with manifest entries 
* /manifests/external - all other projects

Also see domain-model PR - changes need to be added to all relevant projects.

### CODE REVIEW

**Comment by oehm-smith** [comment]
@lbodor updated with changes as discussed

**Comment by oehm-smith** [comment]
Updated to convert GMT date of build-date in MANIFEST to local time zone.

**Comment by oehm-smith** [comment]
Updated to retrieve manifest for the (currently running) Web App.

I'm happy with this now though I'm missing unit tests for this new function.  Just a bit tricky to do.

File: src/main/java/au/gov/ga/geodesy/java/util/service/ManifestService.java [hunk 1]
```diff
@@ -108,11 +111,35 @@ private void buildManifestInfo(Manifest manifest) {
...
         }
 
         /**
+         * Opportunity to change values, such as force dates into the local timezone
+         *
+         * @param key   - from the manifest
+         * @param value - value from the manifest that can be modified
+         * @return the modified (if at all) Value
+         */
+        private String modifyAsAppropriate(String key, String value) {
```
Comment (heyagosper) [inline src/main/java/au/gov/ga/geodesy/java/util/service/ManifestService.java]: Not happy with this method name. It could do anything!

Comment (oehm-smith) [inline src/main/java/au/gov/ga/geodesy/java/util/service/ManifestService.java]: Changed to `updateManifest()`

**Pull request merged**